src/Okta.Sdk.UnitTests/LinkedObjectClientShould.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NSubstitute;
+using Okta.Sdk.Internal;
+using Okta.Sdk.UnitTests.Internal;
+using Xunit;
+
+namespace Okta.Sdk.UnitTests
+{
+    public class LinkedObjectClientShould
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+
+        [Fact]
+        public async Task CallDeleteEndpoint()
```

Comment:
Can `DeleteLinkedObjectDefinitionAsync` be tested via IT instead?

Reply:
Due to server race conditions I was planning on doing the integration test for this after addressing some of the outstanding concerns.